pkgs/tools/misc/vimv/default.nix

```diff
@@ -1,24 +1,27 @@
-{ lib, stdenv, bash, fetchFromGitHub }:
-stdenv.mkDerivation {
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation {
```

Comment:
When should stdenvNoCC be used?

Reply:
I checked the source and there is no invocation of a compiler, it's just a shell script. See https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/misc/neofetch/default.nix for instance.